src/main/resources/messages/LibertyBundles.properties

```diff
@@ -122,3 +122,16 @@ run.config.liberty.project.tool.tip=Select the build file for your Liberty proje
 run.config.start.parameters=Start Parameters
 run.config.start.parameters.tool.tip=eg. -DhotTests=true (Maven) --hotTests=true (Gradle)
 specify.custom.parameters.for.the.mvn.liberty.dev.command=Specify custom parameters for the mvn liberty:dev command
+
+# Maven and Gradle exceptions
+maven.wrapper.does.not.exist=A Maven wrapper for the project could not be found. Make sure to configure a valid Maven wrapper or change the build preferences for Maven inside IntelliJ Maven preferences.
+maven.wrapper.cannot.execute=Could not execute Maven wrapper because the process does not have permission to execute it. Consider giving executable permission for the Maven wrapper file or changing the build preferences for Maven inside IntelliJ Maven preferences.
+maven.invalid.build.preference=Make sure to configure a valid path for Maven home path inside IntelliJ Maven preferences.
+maven.cannot.execute=Could not execute Maven from {0} because the process does not have permission to execute it. Consider giving executable permission for the Maven file or configure IntelliJ to use the Maven wrapper.
```

Comment:
```suggestion
maven.cannot.execute=Could not execute Maven from {0} because the process does not have permission to execute it. Consider giving executable permission for the Maven executable or configure IntelliJ to use the Maven wrapper.
```